lib/jsonapi/include_directive.rb

```diff
@@ -15,8 +15,10 @@ class IncludeDirective
     # @param include_args (see Parser.parse_include_args)
     def initialize(include_args, options = {})
       include_hash = Parser.parse_include_args(include_args)
+
       @hash = include_hash.each_with_object({}) do |(key, value), hash|
-        hash[key] = self.class.new(value, options)
+        validate(key)
```

Comment:
pulled up in 046cb68 (style wise feeling slightly uncomfortable when the method takes two arguments 😅, but I guess in this case it's relatively simple to follow )